site/source/tutorials/003_creating_widgets/index.md

```diff
@@ -142,7 +141,7 @@ export default class Worker extends WidgetBase {
 
 This is nearly identical to the `App` widget with one exception: we are not importing the `w` function as the `Worker` widget will not contain any child widgets.
 
-Our next step is to extend the `render()` method to customize the widget's appearance. To accomplish this, we are going to need two children. One `<img>` tag to show the worker's portrait and a `<strong>` tag to display the worker's name.
+Our next step is to override the `render()` method to customize the widget's appearance. To accomplish this, we are going to need two children. One `<img>` tag to show the worker's portrait and a `<strong>` tag to display the worker's name.
 
 {% instruction 'Try and implement that using the URL `https://dojo.io/tutorials/resources/worker.svg` and `"lastName, firstName"` as the worker\'s name.' %}
 
```

Comment:
Indentation is off in the next solution... (GH wasn't letting me select the line number for the comment as it hasn't been changed

![screenshot 2018-03-04 07 14 13](https://user-images.githubusercontent.com/97291/36946545-a8a01de2-1f7b-11e8-8524-894498a43ba8.png)